miso.cabal

```diff
@@ -200,6 +200,7 @@ library
     base          < 5,
     bytestring    < 0.13,
     containers    < 0.9,
+    generics-sop  < 0.6,
```

Comment:
👀

Reply:
Have switched for `sop-core`. See https://github.com/dmjio/miso/pull/888/commits/f38df9c2b757791f502c5b38b134a0a9fc33ad62.